src/examples/src/config.tsx

```diff
@@ -1665,31 +1666,46 @@ export const config = {
 			overview: {
 				example: {
 					filename: 'Basic',
-					module: BasicSpeedDial
+					module: BasicSpeedDial,
+					sandbox: true,
+					size: 'medium'
```

Comment:
Applicable `SpeedDial` examples were sandboxed so that fixed positions work correctly. They were previously broken and / or used hardcoded widths and heights.